surveys/templates/survey_list_edit.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'partials/large_card.html' %}
+{% load static %}
+
+{% block extracss %}
+  <link rel="stylesheet" href="{% static 'css/dataTables.bootstrap4.min.css' %}">
+{% endblock %}
+
+{% block extrajs %}
+  <script type="text/javascript" src="{% static 'js/jquery.dataTables.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'js/dataTables.bootstrap4.min.js' %}"></script>
+{% endblock %}
+
+{% block head %}Edit Survey{% endblock %}
+
+{% block top_button %}
+    <a class="btn btn-primary ml-sm-auto ml-2 mr-sm-3" href="{% url 'surveys-create' %}">Create new survey</a>
+{% endblock %}
+
+{% block subhead %}
+Use this page to edit survey questions and properties and preview surveys before publishing.
+When a survey is ready to be run it can be published, at which point it can no longer be edited.
+{% endblock %}
+
+{% block card_content %}
+  <table id="survey-list-edit" class="table table-hover table-striped table-responsive-sm card-text">
+  <thead>
+    <tr>
+      <th scope="col">Survey Name</th>
+      <th scope="col">Last Updated</th>
+      <th scope="col">Type</th>
+      <th class="text-center" scope="col">Edit</th>
+      <th class="text-center" scope="col">Preview</th>
+      <th class="text-center" scope="col">Publish</th>
+      <th class="text-center" scope="col">Delete</th>
```

Comment:
I like the new styles for these icons a lot! The columns are still labelled with their specific actions, however, instead of being grouped under one heading for `Actions`:

![Screen Shot 2019-06-11 at 5 23 26 PM](https://user-images.githubusercontent.com/14170650/59360116-2e1c6600-8cf5-11e9-90f7-9cb114af2ffb.png)

Was this what you were intending? Maybe we could have a quick convo IRL to talk about ways of condensing these actions.

Reply:
I opened a new issue for that here: #168 

I started to condense them into one column and couldn't find a way that didn't feel pretty cluttered and confusing. Do you know of any example sites I could reference?

Reply:
Cool! I'm fine pushing off to #168 since you tried a few options and didn't like them as much.